chore(tos): Update docs + image for legal and compliance (#9986)
getsentry/sentry-docs: docs/security-legal-pii/security/terms.mdx

---
title: "Legal and Compliance Documents"
sidebar_order: 1

> description: "Learn about Sentry's Terms of Service, Privacy Policy, Data Processing Addendum, and Business Associate Amendment. Sentry also provides a way to self-serve most legal and compliance documentation for your organization."

Reviewer: ```suggestion
description: "Learn about Sentry's Terms of Service, Privacy Policy, Data Processing Addendum, and Business Associate Amendment."
```
Removing this since this is the next sentence in the docs :)

---

> Sentry provides a way to self-serve most legal and compliance documentation for your organization. You'll find these by navigating to _Your Organization's Settings_ > Legal & Compliance. When you are logged in to your Sentry account, you will be able to review or accept these agreements as an authorized individual on behalf of your organization.

Reviewer: ```suggestion
You can self-serve most legal and compliance documentation for your organization by navigating to _Your Organization's Settings_ > Legal & Compliance. When you're logged into your Sentry account, you'll be able to review or accept these agreements as an authorized individual on behalf of your organization.
```
